### DISCUSSION TITLE
[Office Hours] Merge Conflict

### DISCUSSION THREAD

**Discussion by jennxu23** [CS342/.github#24] [Office Hours] [answered]
### Related Discussion

Hi! I'm trying to merge my branch into the main branch, and I having difficulties. I think I didn't resolve all the merge conflict, or accidentally left some of the `===<<<` markers somewhere, but I can't really tell because I can't open and build my file in XCode. 

### Reproduction

I tried to resolve merge conflicts in github in the online editor. Then, I updated my local branch. I was able to open the files on my local branch and fix a couple mistakes, but I couldn't build and now I can't open the file all together.

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by PSchmiedmayer** [comment, marked as answer]
@jennxu23 Sounds good, we can take a look at that tomorrow during the office hours 👍
